src/test/resources/roles_system_indices.yml

```diff
@@ -12,3 +12,25 @@ opendistro_security_all_access:
         - '*'
       allowed_actions:
         - '*'
+
+extensions_role:
+  description: "ExtensionsRole"
+  cluster_permissions:
+    - '*'
+  index_permissions:
+    - index_patterns:
+        - '*'
+      allowed_actions:
+        - '*'
+        - 'system:admin/system_index'
```

Comment:
If the index pattern was `.k*` would only system indices that start with the letter k be allowed, so `.alerting` would be denied access, but `.knn` would be allowed?  Can we get a test that confirms this?